src/lwc/geFormWidget/geFormWidget.js

```diff
@@ -11,36 +12,58 @@ const WIDGET_LIST = [PAYMENT_SCHEDULER_WIDGET, ALLOCATION_WIDGET, TOKENIZE_CARD_
 
 export default class GeFormWidget extends LightningElement {
     @api element;
-    @api widgetData;
+    @api widgetConfig;
 
     @track widgetDataFromState = {};
 
     _formState = {};
+
+    _allocationFields = [apiNameFor(DI_DONATION_AMOUNT_FIELD), apiNameFor(DI_ADDITIONAL_OBJECT_JSON_FIELD)];
+    _elevateFields = [apiNameFor(DATA_IMPORT_PAYMENT_METHOD)];
+
+    get sourceFieldsUsedInTemplate() {
+        return this.widgetConfig ? this.widgetConfig.sourceFieldsUsedInTemplate : [];
```

Comment:
That would totally work! My thinking here was to guard against referencing undefined in the case that widgetConfig/widgetConfig.sourceFieldsUsedInTemplate wasn't provided. Any logic that depends on finding a field in the sourceFieldsUsedInTemplate array could then behave as though the field wasn't included in the template instead of attempting to read an undefined property. What do you think?

Reply:
I generally prefer returning empty arrays / lists when things are not found as a result of a search.

Reply:
👍🏽